bencher/test/src/lib.rs

```diff
@@ -0,0 +1,84 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+pub mod benches;
+pub mod mock;
+mod tests;
+mod weights;
+
+#[frame_support::pallet]
+pub mod pallet {
+	use crate::weights::ModuleWeights;
+	use frame_support::{
+		dispatch::{DispatchResult, DispatchResultWithPostInfo},
+		pallet_prelude::*,
+		transactional,
+	};
+	use frame_system::pallet_prelude::*;
+
+	#[pallet::config]
+	pub trait Config: frame_system::Config {}
+
+	#[pallet::hooks]
+	impl<T: Config> Hooks<T::BlockNumber> for Pallet<T> {}
```

Comment:
No need for empty hooks and config fields... Just semantics not important at all